### PR TITLE
Allow compact-migrations to work with HTTP

### PR DESCRIPTION
**Pull request by apmasell**
The URL it was detecting was specific to SSH. This URL detection will match both

### CODE REVIEW

File: compact-migrations [hunk 1]
```diff
@@ -34,6 +34,7 @@ fi
 
 NEXT="V$(printf "%04d" ${ID})__auto_${TAIL}.sql"
 echo "Next migration will be ${NEXT}."
+exit 1
```
Comment (djcooke) [inline compact-migrations]: I'm guessing this was for testing and should be removed now?

Comment (apmasell) [inline compact-migrations]: Yes, fixed.

**Comment by djcooke** [comment]
:+1:

**Comment by tonydebat** [comment]
👍

**Pull request merged**